app/routines/update_user_salesforce_info.rb

```diff
@@ -54,6 +54,7 @@ def call
         cache_contact_and_school_data_in_user!(contact, school, user)
       rescue StandardError => ee
         error!(exception: ee, user: user)
+        Sentry.capture_exception ee
```

Comment:
It looks like you are adding this everywhere that `error` is called. You could probably just call this one time in this function: https://github.com/openstax/accounts/blob/prevent-salesforce-errors/app/routines/update_user_salesforce_info.rb#L324-L336

Reply:
@mwharrison I was selective about where I added it because not all of the errors collected are exceptions and should not be sent to Sentry

Reply:
Ok - approving